integration/test_synchronize.py

```diff
@@ -25,6 +25,14 @@
     twisted_sleep,
 )
 
+def non_lit_content(s):
+    # type: (unicode) -> unicode
+    """
+    Pad the given string so it is long enough to not fit in a tahoe literal
+    URI.
+    """
+    return "{} {}\n".format(s, "." * max(55 - len(s), 0))
```

Comment:
I'd greatly prefer to not have a magic-number here .. but of course I'd _also_ like to not have to import `allmydata.*` code .. can you at least document where this comes from so that later authors can confirm its still valid.

Reply:
Given that the constant we'd need to import is `allmydata.immutable.upload.Uploader.URI_LIT_SIZE_THRESHOLD`, I definitely think we shouldn't import it. I've left a comment (and included the name of the constant).